.circleci/config.yml

```diff
@@ -170,16 +170,17 @@ workflows:
           filters:
             branches:
               only: master
-  uat:
-    triggers:
-      - schedule:
-          cron: "0 11 * * *"
-          filters:
-            branches:
-              only:
-                - develop
-                - main
-    jobs:
-      - test
+
+  # uat:
+    # triggers:
+    #   - schedule:
+    #       cron: "0 11 * * *"
+    #       filters:
+    #         branches:
+    #           only:
+    #             - develop
+    #             - main
+    # jobs:
+    #   - test
```

Comment:
Keep the test line too, so it will be run in case of PR. Just comment 173-183 included.

Reply:
The tests are already run on each commit, if you look line 152 in the workflow build_test_package